src/main/resources/hudson/security/LDAPBindSecurityRealm.groovy

```diff
@@ -65,10 +65,11 @@ bindAuthenticator(BindAuthenticator2,initialDirContextFactory) {
     userSearch = ldapUserSearch;
 }
 
-authoritiesPopulator(AuthoritiesPopulatorImpl, initialDirContextFactory, instance.groupSearchBase) {
+authoritiesPopulator(AuthoritiesPopulatorImpl, initialDirContextFactory, instance.groupSearchBase, instance.groupMembershipStrategy) {
     // see DefaultLdapAuthoritiesPopulator for other possible configurations
     searchSubtree = true;
     groupSearchFilter = "(| (member={0}) (uniqueMember={0}) (memberUid={1}))";
+    ldapSearch = ldapUserSearch;
```

Comment:
> does it still make sense to be overwritten? 

No, but there are instances that did override it back when that was the only way, and this plugin has to stay working for those people... OTOH you can assume that they do not have fancy configurations in that case